File: easy/1317_convert_integer_to_the_sum_of_two_no_zero_integers/solution.py
from typing import List


class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        a, b = 1, n - 1

        while "0" in str(a) + str(b):
            a += 1
            b -= 1

        return [a, b]

File: easy/1317_convert_integer_to_the_sum_of_two_no_zero_integers/solution.cpp
#include <vector>

class Solution {
public:
  bool containsZero(int n) {
    while (n > 0) {
      if (n % 10 == 0)
        return true;
      n /= 10;
    }

    return false;
  }

  std::vector<int> getNoZeroIntegers(int n) {
    for (int a = 1, b = n - 1;; a++, b--) {
      if (!containsZero(a) && !containsZero(b))
        return {a, b};
    }

    return {};
  }
};
